### PR TITLE
Add typescript, node, and AWS services

### PR DESCRIPTION
**Pull request by umarov**

### CODE REVIEW

**Review by ryanbales — COMMENTED** [review]
Thanks @umarov   Nice work overall, do you mind bumping those packages that the Security Scanner suggested?

**Comment by umarov** [comment]
@ryanbales I got rid of htmllint. it wasn't even working. Updated everything to latest. Looks like they are for the ride from the fork

**Review by ryanbales — APPROVED** [review]
LGTM 👍 Thanks for updating the tech radar @umarov !

File: docs/config.json [hunk 1]
```diff
@@ -741,9 +773,9 @@
...
       "timeline": [
         {
-          "moved": 0,
-          "ringId": "assess",
-          "date": "2022-05-01",
+          "moved": 1,
+          "ringId": "trial",
+          "date": "2023-04-01",
           "description": ""
         }
```
Comment (nburwell) [inline docs/config.json:780]: minor, but I believe the intention is to keep adding to the timeline, versus updating it. i.e. would look like this:

```
"timeline": [
  {
    "moved": 0,
    "ringId": "assess",
    "date": "2022-05-01",
    "description": ""
  },
  {
     "moved": 1,
     "ringId": "trial",
     "date": "2023-04-01",
     "description": "Node is now being used in production: Recording Proxy Service"
   }
```

**Review by nburwell — COMMENTED** [review]
Thanks for updating this!

File: docs/config.json (hunk 1 above)
Comment (umarov) [reply docs/config.json:780]: ahhh. I can make another PR to fix it.

**Pull request merged**